app/routes/languages.rb

```diff
@@ -70,6 +70,11 @@ def topic_not_found(topic)
         status 404
         erb :"languages/not_found", locals: { track_id: topic }
       end
+
+      def active(topic, selected_topic)
+        'active' if topic == selected_topic
+      end
```

Comment:
This would be better if it was defined in the the erb file, since it's entirely related to the display.